htdocs/categories/class/categorie.class.php

```diff
@@ -2010,4 +2010,28 @@ public static function getFilterSelectQuery($type, $rowIdName, $searchList)
 			return "";
 		}
 	}
+
+	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps
+	/**
+	 *      Count all categories
+	 *
+	 *      @return int                             Number of categories, -1 on error
+	 */
+	public function count_all_categories()
```

Comment:
For new functions, we must now follow the CamelCase for naming https://en.wikipedia.org/wiki/Camel_case 
Can you rename it into 
countNbOfCategories()